source/time_tables/passages.cpp

```diff
@@ -164,6 +164,9 @@ void passages(PbCreator& pb_creator,
         } else {
             passage = pb_creator.add_next_arrivals();
         }
+
+        // We need one second before because else, on terminus, we may
+        // miss something as period exclude the end.
```

Comment:
minor: maybe precise that `disruption` period in particular excludes the end, here...